src/core/CMakeLists.txt

```diff
@@ -110,10 +110,26 @@ SET(QFIELD_CORE_HDRS
   expressionevaluator.h
 )
 
-FIND_PACKAGE(Qt5 COMPONENTS Qml Core Gui Xml Positioning Widgets PrintSupport Network Quick REQUIRED)
+IF (ANDROID)
+  SET(QFIELD_CORE_HDRS ${QFIELD_CORE_HDRS}
+    androidplatformutilities.h
+    androidpicturesource.h
+    androidviewstatus.h
+    androidprojectsource.h
```

Comment:
not strictly related to this PR, but it would be nice to group these under an android folder